docs/ext/ipc/client.rst

```diff
@@ -1,17 +1,16 @@
-Client Connection
-=================
-
-The IPC client is very simple.
-It will simply connect to your server process and send JSON data.
-If you do not supply a port on initialisation, the client will connect to the multicast server
-(see the server section) and return the port from said server.
-If you do supply a port, it will connect to the server instantly.
-
-Requests are made by calling ``ipc.client.request(endpoint, **kwargs)``
-and will be sent to the server in the json format specified above.
-It will then wait for a response and return the data.
-
-.. currentmodule:: nextcord.ext.ipc.client
-
-.. autoclass:: Client
-    :members:
+Client Connection
+=================
+
+The IPC client is very simple.
+It will simply connect to your server process and send JSON data.
+If you do not supply a port on initialisation, the client will connect to the multicast :doc:`server` and return the port from said server.

```

Comment:
Link text shows up as "Server Setup", should be "server"

![image](https://user-images.githubusercontent.com/20955511/139822364-06932c0f-4875-47c1-b96a-6569273a29c5.png)

```suggestion
If you do not supply a port on initialisation, the client will connect to the multicast :doc:`server<server>` and return the port from said server.

```